apps/prairielearn/src/pages/instructorAssessmentAccess/instructorAssessmentAccess.ts

```diff
@@ -19,7 +28,95 @@ router.get(
       { assessment_id: res.locals.assessment.id },
       AssessmentAccessRulesSchema,
     );
-    res.send(InstructorAssessmentAccess({ resLocals: res.locals, accessRules }));
+
+    const assessmentPath = path.join(
+      res.locals.course.path,
+      'courseInstances',
+      res.locals.course_instance.short_name,
+      'assessments',
+      res.locals.assessment.tid,
+      'infoAssessment.json',
+    );
+
+    const assessmentPathExists = await fs.pathExists(assessmentPath);
+
+    let origHash = '';
+    if (assessmentPathExists) {
+      origHash = sha256(b64EncodeUnicode(await fs.readFile(assessmentPath, 'utf8'))).toString();
+    }
+
+    const now = Temporal.Now.zonedDateTimeISO(res.locals.course_instance.timezone);
+    const formatter = new Intl.DateTimeFormat('en-US', {
+      timeZone: res.locals.course_instance.display_timezone,
+      timeZoneName: 'short',
+    });
+    const timezone = formatter
+      .formatToParts(now.toInstant().epochMilliseconds)
+      .find((part) => part.type === 'timeZoneName')?.value;
```

Comment:
What's going on here? Is there a reason you aren't using `course_instance.timezone` directly?

Reply:
I was using this to construct the timezone "shortname" (such as 'CDT' or 'PST) and using this for display in the modal. However, after discussing we have decided that PrairieTest uses the full timezone name ("America/Chicago") on time inputs so I have updated here to do the same. This resulted in removing these lines and using the `course_instance.timezone` directly.